### DISCUSSION TITLE
mrgsolve blog

### DISCUSSION THREAD

**Discussion by Mutaz94** [metrumresearchgroup/mrgsolve#1084] [General]
Hi! I want to drop that your blog is great and very informative. While going through different articles, this one got my attention and I couldn't follow the idea behind your notes about modeling the duration. 
https://mrgsolve.org/blog/posts/zero-ss.html

Do you mean we don't try to model them together, or we should no model duration?

Thanks! 
Mutaz

**Comment by kylebaron** [comment]
Hi @Mutaz94  - 

The idea is you can have two types of infusions at steady-state: intermittent infusions (100 mg infused over 30 min every 12 hours, repeated to steady state) or this non-intermittent infusion (I turned on an infusion that never ends and eventually the concentration will make it's way to steady state). That blog article talks about this non-intermittent infusion at steady state and in that case, there is no duration or dosing interval; the system just makes its way to steady state _with this zero-order input always going_. 

Does that make sense?

Kyle

**Reply by Mutaz94** [reply]
Ahhh, that is a nice way to obtain `ss`. Yes, it make sense now. Thank you again for these articles.
